@kiva/kv-components/vue/KvExpandableQuestion.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<div>
+		<button
+			class="tw-w-full tw-py-2 tw-flex tw-justify-between"
+			@click="toggleFAQ"
+		>
+			<h3 class="tw-text-subhead tw-text-left">
+				{{ title }}
+			</h3>
+			<kv-material-icon
+				class="tw-w-4 tw-h-4"
+				:icon="open ? mdiChevronUp : mdiChevronDown"
+				@click="open = !open"
+			/>
+		</button>
+		<kv-expandable easing="ease-in-out">
+			<div
+				v-show="open"
+				class="tw-prose tw-pb-4 tw-pt-2"
+				v-html="content"
+			>
+			</div>
+		</kv-expandable>
+	</div>
+</template>
+
+<script>
+import {
+	computed,
+	onMounted,
+	watch,
+	ref,
+	toRefs,
+} from 'vue-demi';
+import {
+	mdiChevronDown,
+	mdiChevronUp,
+} from '@mdi/js';
+import { paramCase } from 'change-case';
+
+import KvExpandable from './KvExpandable.vue';
+import KvMaterialIcon from './KvMaterialIcon.vue';
+
+export default {
+	components: {
+		KvMaterialIcon,
+		KvExpandable,
+	},
+	props: {
+		/**
+		 * Question Title
+		* */
+		title: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * Question Content - can accept raw html
+		* */
+		content: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * Analytics Category
+		* */
+		analyticsCategory: {
+			type: String,
+			default: 'FAQ',
+		},
+		active: {
+			type: Boolean,
+			default: false,
+		},
+		routeHash: {
+			type: String,
+			default: '',
+		},
+		kvTrackFunction: {
+			type: Function,
+			default: () => {},
+		},
+	},
+	emits: [
+		'toggle',
+	],
+	setup(props, { emit }) {
+		const {
+			title,
+			active,
+			analyticsCategory,
+		} = toRefs(props);
+		const open = ref(active.value || false);
+		const titleSlugified = computed(() => paramCase(title.value));
+
+		const toggleFAQ = () => {
+			if (!open.value) {
+				props.kvTrackFunction(analyticsCategory.value, 'click-faq-expand', title.value);
```

Comment:
The category should be `faq`
The action should be `toggle`
The label can be the titleSlugified
The property can be used to indicate the state of the question maybe `expand` or `shrink`

So this can be `kvTrackFunction(`faq`, 'toggle', titleSlugified, <expand or shrink>`

Reply:
Cool I think that makes sense. 

@tiarascoleman you should be able to assign that last property conditionally based on the value of `open.value` with a ternary operator.

Also if we're going to track both the expand and shrink states, we should remove the `if (!open.value) {}` block around the tracking call.